_pages/security/dynamic-scanning.md

```diff
@@ -11,6 +11,10 @@ parent: Security
 
 You will need a running application to test, which you will want to be as production-like as possible. The 'Attack' scripts shouldn't do anything destructive, but will cause a spike in requests and errors. **Inform your team and #devops before running it on live (not locally-hosted) sites.**
 
+We gave an introduction to ZAP talk as part of our engineering tech talks series. [Vulnerability with OWASP ZAP](https://www.youtube.com/watch?v=2Dp7pAvKHaM) ([more info and slides](https://github.com/18F/tech-talks/tree/master/vuln-scanning))
```

Comment:
Not a blocker, but I would use the preview image syntax to give this a bigger call-out.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#videos